lib/copy/ncp.js

```diff
@@ -89,6 +81,11 @@ function ncp (source, dest, options, callback) {
             copyFile(file, target)
           })
         } else {
+          var err = new Error('EEXIST: ' + target + ' already exists.')
+          err.code = 'EEXIST'
+          err.errno = -17
+          err.path = target
+          onError(err)
```

Comment:
@hhamilto Good catch! I meant to remove the `doneOne` call below, but somehow missed it.